catalyst/utils/quantization.py

```diff
@@ -14,13 +14,64 @@
     unpack_checkpoint,
 )
 
+from catalyst.utils.onnx import convert_to_onnx
+
+from catalyst.settings import IS_ONNX_AVAILABLE
+
+if IS_ONNX_AVAILABLE:
+    from catalyst.utils.onnx import quantize_onnx_model
+
 if TYPE_CHECKING:
     from catalyst.experiments import ConfigExperiment
 
 
 logger = logging.getLogger(__name__)
 
 
+def quantize(
```

Comment:
speaking about the naming,
should we use something like
```
- trace_model
- prune_model
- quantize_model
```
for user-friendly API?